### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by DevAseel**
This PR is meant to prepare for the 0.3.0 release, and contains the following: 
- New generated typedocs.
- Cleaned up examples.
- Cleaned up tests.
- New documentation.

### CODE REVIEW

File: docs/library/queries.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
+`blocks`
```
Comment (specialistvlad) [inline docs/library/queries.md:1]: let's have it as a sql files

**Comment by specialistvlad** [comment]
@DevAseel what about README.md file in the root of the folder? Do we have reference there to new documentation? Will we do there some modifications?

File: docs/library/queries.md (hunk 1 above)
Comment (DevAseel) [reply docs/library/queries.md]: Done!

**Comment by DevAseel** [comment]
> @DevAseel what about README.md file in the root of the folder? Do we have reference there to new documentation? Will we do there some modifications?

@specialistvlad I think it's better to discuss with @jjfluid, also in regards with `change.log` file.

**Review by mrrfid — COMMENTED** [review]
The references in the readme.md to the other documentation makes it much cleaner. Look good to me.

**Review by mrrfid — APPROVED** [review]
Looks good. Very complete documentation.

**Pull request merged**